packages/barqode/src/lib/index.ts

```diff
@@ -1,12 +1,12 @@
 export { default as BarqodeCapture } from "./components/barqode-capture.svelte";
-export { default as BarqodeDropZone } from "./components/barqode-drop-zone.svelte";
+export { default as BarqodeDropzone } from "./components/barqode-dropzone.svelte";
 export { default as BarqodeStream } from "./components/barqode-stream.svelte";
 
 export type {
 	DetectedBarcode,
 	BarcodeFormat,
 	Point2D,
 	CaptureProps,
-	DropZoneProps,
+	DropzoneProps as DropZoneProps,
```

Comment:
I don't know if I'm tired but this makes me super confused.

we changed `DropZoneProps` to `DropzoneProps` everywhere.

why are we then exporting `DropzoneProps as DropZoneProps`

and if we do that, how did this even pass linting? 🤔

Reply:
Yeah, def  not intended! Nice catch! We passed lint/check since we're not using that type anywhere in the docs nor the export from `lib/index.ts` anywhere in the components (which is a good thing to avoid circular dep hell)

Reply:
ah right, makes sense!